CUDA/src/gpuNUFFT_operator_factory.cpp

```diff
@@ -420,7 +420,7 @@ gpuNUFFT::Array<DType> gpuNUFFT::GpuNUFFTOperatorFactory::computeDeapodizationFu
 
   DType maxDeapoVal = 0;
   DType minDeapoVal = std::numeric_limits<DType>::max();
-  double fft_scaling_factor = std::sqrt(imgDims.count()); 
+  double fft_scaling_factor = std::sqrt(deapoGpuNUFFTOp->getGridDims().count()); 
```

Comment:
You are using an already deleted pointer `deapoGpuNUFFTOp` here -> see line 415. Please move one of the statements accordingly.